weather_sp/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 global-exclude *_test.py
 include README.md
-prune test_data
+# Removing this line to fix the warning message issue #243
+# prune test_data
```

Comment:
On second though, let's delete this instead of leave a trace of it. Further, will this include `test_data` in the packaging? If so, how do we make our package/binary size smaller?